blog/2024/08/starting-workgroups.md

```diff
@@ -0,0 +1,64 @@
+@def title = "So, You Want to Start a Julia Workgroup?"
+@def authors = "Hetarth Shah, George Datseris, Jacob Zelko"
+<!--
+TODO: Add this last
+   - @def published = "3 May 2024"
+   - @def rss_pubdate = Date(2024, 5, 3)
+   -->
+@def rss = """How you can start your own Julia community workgroup."""
+
+## Introduction
+
+Hey there, this post is dedicated to all those people who want to start a working group in julia. If you already have one, there might be some helpful tips for you too.
+
+#### What is a working group?
+
+As per [Julia's official website](https://julialang.org/community/working-groups/#working_groups),
+
+> A working group is a group of people that are interested in working on a common topic. Working groups are informal and have no official authority. All working groups have a public channel on the Julia Slack.
+
+Not only this but most working groups regularly hold meetings, everyone is free to attend these meetings and discuss about the topics which they want to. Most meeting schedules are posted on julia's community events calender but one can also find relevant links in the respective slack channels.
+
+Alright, now that we know what a working group is.
+
+#### Gaining Members
+
+> You created a working group now what?
```

Comment:
So far there are no instructions on how to create such a group. Moreover, I would assume that creating the group and gaining at least its founding/starting members occur simultaneously. So I would recommend to have a section on "starting a group" that discusses also how to invite members.